--- bfd-in2.h
+++ bfd-in2.h	2012-07-17 16:42:55.000000000 +0200
@@ -38,6 +38,8 @@
 
 #include "ansidecl.h"
 #include "symcat.h"
+#include <sys/stat.h>
+
 #if defined (__STDC__) || defined (ALMOST_STDC) || defined (HAVE_STRINGIZE)
 #ifndef SABER
 /* This hack is to avoid a problem with some strict ANSI C preprocessors.
@@ -308,9 +310,6 @@
    && (sec)->sec_info_type != ELF_INFO_TYPE_MERGE		\
    && (sec)->sec_info_type != ELF_INFO_TYPE_JUST_SYMS)
 
-/* Forward define.  */
-struct stat;
-
 typedef enum bfd_print_symbol
 {
   bfd_print_symbol_name,
